src/sources/gdal.jl

```diff
@@ -277,9 +277,9 @@ metadata(raster::AG.RasterDataset, args...) = begin
     offset = AG.getoffset(band)
     # norvw = AG.noverview(band)
     units = AG.getunittype(band)
-    path = first(AG.filelist(raster))
+    # path = first(AG.filelist(raster))
     meta = AG.metadata(raster.ds)
-    GDALarrayMetadata(Dict("filepath"=>path, "scale"=>scale, "offset"=>offset, "units"=>units))
+    GDALarrayMetadata(Dict("scale"=>scale, "offset"=>offset, "units"=>units))
```

Comment:
This could be put back to how it was before since I abandoned efforts to convert between `ArchGDAL.Dataset and `GDALarray`